testbed/HIL_Firmware/VCU_HIL/VCU_HIL/userInit.c

```diff
@@ -0,0 +1,184 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/queue.h"
+#include "esp_err.h"
+#include "driver/twai.h"
+#include "driver/spi_master.h"
+#include "userInit.h"
+#include "dac.h"
+#include "canReceive.h"
+#include "processCAN.h"
+
+spi_device_handle_t throttle_A;
+spi_device_handle_t throttle_B;
+spi_device_handle_t brake_pos;
+spi_device_handle_t steer_raw;
+
+void taskRegister (void)
+{
+    BaseType_t xReturned = pdPASS;
+    TaskHandle_t can_rx = NULL;
```

Comment:
Didnt look very hard but seems sus that you are setting this to null then passing the variable by reference on line 32. I'm sure it works but do you have a link that I can reference for future? The only reason you'd do smth like this is if you are trying to allocate to the heap which maybe xTaskCreate does but also that may be why theres a limit on the number of tasks we can have bc we really shouldn't have a big heap/ allocating to it at all

Reply:
I figured it would make sense to init the handle to NULL until it gets set with xTaskCreate, will remove